src/js/providers/tracks-mixin.ts

```diff
@@ -324,8 +324,8 @@ const Tracks: TracksMixin = {
                     } else {
                         trackId = track._id = createId(track, this._textTracks ? this._textTracks.length : 0) as string;
                     }
-                    if (_tracksById[trackId]) {
-                        // tracks without unique ids must not be marked as "inuse"
+                    if (_tracksById[trackId] || (this.renderNatively && track.mode === 'disabled')) {
+                        // tracks without unique ids must not be marked as "inuse", unless they are natively renddered and explicitly set to not "disabled"
```

Comment:
Typos: "renddered", set to not "disabled"?

Reply:
Make sure this isn't an issue with metadata tracks either.

`(this.renderNatively && track.mode === 'disabled')`
This requires testing in Safari and Chrome, on all three providers/media types (hls, dash, mp4) both with and without `renderCaptionsNatively` enabled.